src/main/java/org/jenkinsci/plugins/mesos/api/LaunchCommandBuilder.java

```diff
@@ -137,13 +137,19 @@ public LaunchPod build() throws MalformedURLException, URISyntaxException {
 
   /** @return the agent shell command for the Mesos task. */
   private String buildCommand() throws MalformedURLException {
-    return String.format(
-        AGENT_COMMAND_FORMAT,
-        this.xmx,
-        this.jvmArgString,
-        this.jnlpArgString,
-        buildJnlpSecret(),
-        buildJnlpUrl());
+    // Check if command is overriden
```

Comment:
Please see `SimpleTaskInfoBuilder` in USI. The `shellCommand` will call 
```scala
      taskInfoBuilder.setCommand(
        Mesos.CommandInfo
          .newBuilder()
          .setShell(true)
          .setValue(shellCommand)
```
this is different from [1.x](https://github.com/jenkinsci/mesos-plugin/blob/1.x/src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java#L1050) where we have
```java
  commandBuilder.setShell(false);
  commandBuilder.setValue(customShell);
``` 
Kai, do you know why we set shell to `false`?

Reply:
shell is set to `true` if we want to pass a shell script to be evaluated by `sh`; `setShell` effectively should've been called `setShellScript(true)`.